### PR TITLE
Deprecated source function issue #14908

### PR DESCRIPTION
**Pull request by udiptya999**
Added deprecated decorator to source function

#### Brief description of what is fixed or changed
Deprecated decorator added

### CODE REVIEW

**Comment by udiptya999** [comment]
@asmeurer Sir please review my PR request

**Comment by smichr** [comment]
I am -1/2 to this...I will miss this little function. Must we lose this dash of sugar?

**Comment by asmeurer** [comment]
@smichr what about `inspect.getsource`? This is basically a thin wrapper around it.

**Comment by udiptya999** [comment]
Is it depracate not required here? it returns the source code here I think

File: sympy/utilities/source.py [hunk 1]
```diff
@@ -5,8 +5,9 @@
 from __future__ import print_function, division
 
 import inspect
+from sympy.core.decorators import deprecated
 
-
+@deprecated(issue=14908, deprecated_since_version="1.1")
```
Comment (asmeurer) [inline sympy/utilities/source.py]: The most recent release is 1.2. This should have 1.3.

Comment (asmeurer) [reply sympy/utilities/source.py]: Also, add the useinstead flag.

Comment (udiptya999) [reply sympy/utilities/source.py]: should I change it now?

Comment (udiptya999) [reply sympy/utilities/source.py]: and what to fill in useinstead flag argument?

**Pull request closed without merging**